Ginger/Ginger/AutomatePageLib/AddActionMenu/SharedRepositoryLib/ActivitiesRepositoryPage.xaml.cs

```diff
@@ -143,6 +151,35 @@ private void EditActivity(object sender, RoutedEventArgs e)
             }
         }
 
+        public void ShowAsWindow(Window ownerWindow, eWindowShowStyle windowStyle = eWindowShowStyle.Dialog)
+        {
+            Button addButton = new Button();
```

Comment:
Show as window buttons/other change should be done based on ViewMode value so other dev will be able to show this page as window but control if to have "Ad Selected" or not via the ViewMode

Reply:
Done !